src/main/java/org/wso2/lsp4intellij/editor/EditorEventManager.java

```diff
@@ -834,10 +832,9 @@ private LookupElement createLookupItem(CompletionItem item) {
 
         if (textEdit != null) {
             if (addTextEdits != null) {
-                addTextEdits.add(textEdit);
                 lookupElementBuilder = setInsertHandler(lookupElementBuilder, addTextEdits, command, label);
             } else {
-                lookupElementBuilder = setInsertHandler(lookupElementBuilder, Collections.singletonList(textEdit),
+                lookupElementBuilder = setInsertHandler(lookupElementBuilder, Collections.emptyList(),
```

Comment:
Shall we remove the else block here since we are creating a empty insert handler? :)

Reply:
Removed :+1: